src/Modal/ModalDialogBody.jsx

```diff
@@ -24,7 +24,7 @@ const ModalDialogBody = ({
     (
       <>
         <div ref={topSentinelRef} />
-        <div className="pgn__modal-body-content">
+        <div className="pgn__modal-body-content" style={{ 'z-index': '-1', position: 'relative' }}>
```

Comment:
@abutterworth is it okay to pass inline css or should I create a class for the with these properties and pass that here?

Reply:
Oh yes, sorry see my note below